Core/Src/DynamicControls/lateral_control.c

```diff
@@ -31,8 +31,8 @@ lateral_t lateral_control() {
     float cg_speed;
     double gyro_adjusted;    // entre -1.5 e 1.5
     float steering_adjusted; // entre -0.5 e 0.5
-    double desired_yaw, max_yaw, setpoint;
-    lateral_t ref_torque;
+    double desired_yaw, max_yaw, setpoint, pid_result;
+    lateral_result_t ref_torque_result;
```

Comment:
Inicializa com 0, já que o valor de decremento é só pra uma roda (subentendendo que a outra vai ficar zerada).
Na realidade, é padrão dos compiladores inicializar structs com 0 por default, mas nunca é bom depender de compilador pra comportamentos rs

Reply:
não to conseguindo inicializar de maneira bonita, pelo fato de ser membro de uma struct

Reply:
consegui xD